ledger/error/src/main/scala/com/daml/error/definitions/PackageServiceError.scala

```diff
@@ -19,7 +19,14 @@ import com.daml.lf.data.Ref.PackageId
 import com.daml.lf.engine.Error
 import com.daml.lf.{VersionRange, language, validation}
 
-trait PackageServiceError extends BaseError
+abstract class PackageServiceError(
+    override val cause: String,
+    override val throwableO: Option[Throwable] = None,
+)(implicit override val code: ErrorCode)
+    extends BaseError.Impl(cause, throwableO) {
+  final override def logOnCreation: Boolean = true
```

Comment:
That is a default on BaseError.Impl

Reply:
yes and I want to be sure it stays like that

Reply:
i.e. enforce that `PackageServiceError` logs on creation.